quint/src/generated/Quint.g4

```diff
@@ -167,25 +171,19 @@ identOrStar :   '*' | IDENTIFIER
 path    : IDENTIFIER ('.' IDENTIFIER)*
         ;
 
-argList :      expr (',' expr)*
+argList :      '(' expr? (',' expr)* ')'
         ;
 
 // operators in the normal call may use a few reserved names,
 // which are not recognized as identifiers.
 normalCallName :   IDENTIFIER
-        |       op=(AND | OR | IFF | IMPLIES | SET | LIST | MAP)
+        |       op=('and' | 'or' | 'iff' | 'implies' | SET | LIST | MAP)
```

Comment:
See, now we have duplicate stings on at least 4 different places, and each one is an opportunity for a typo.